lib/re.js

```diff
@@ -56,7 +56,7 @@ module.exports = function (opts) {
           '\\"(?:(?!' + re.src_ZCc + '|["]).)+\\"|' +
           "\\'(?:(?!" + re.src_ZCc + "|[']).)+\\'|" +
           "\\'(?=" + re.src_pseudo_letter + '|[-]).|' +  // allow `I'm_king` if no pair found
-          '\\.{2,4}[a-zA-Z0-9%/]|' + // github has ... in commit range links,
+          '\\.{2,}[a-zA-Z0-9%/]|' + // github has ... in commit range links,
```

Comment:
1. Add note `google has many dots in "google search" links`.
2. Add your example link to tests.